src/app/operators/components/marble-diagram/marble-diagram.component.html

```diff
@@ -1,8 +1,9 @@
 <div class="marble-wrapper mat-elevation-z2">
   <img class="marble-diagram"
        [src]="url"
-       *ngIf="url" />
+       *ngIf="url"
+       alt="Diagram of how combineAll works" />
```

Comment:
I think we need to make a small update so `combineAll` isn't hardcoded, something like:

`alt="Diagram of how {{operatorName}} works"`

Or we can add a getter to this component for the alt text:

```ts
get altText() {
  return `Diagram of how ${this.operatorName} works`;
}
```

then:

`[alt]="altText"`